### ISSUE TITLE
[Bug] PermissionException in UWP although specific DeviceCapability declared

### ISSUE THREAD

**Issue by anders9ustafsson** [xamarin/Essentials#1201]
### Description
Even though I have explicitly set e.g. the _microphone_ capability in the _Package.appxmanifest_ file of my UWP app, I still get the following exception when trying to check permission status:

>Xamarin.Essentials.PermissionException
You need to declare the capability `microphone` in your AppxManifest.xml file
   at Xamarin.Essentials.Permissions.BasePlatformPermission.EnsureDeclared()
   at Xamarin.Essentials.Permissions.BasePlatformPermission.CheckStatusAsync()

### Steps to Reproduce

1. Declare _microphone_ capability in _Package.appxmanifest_ file of a UWP project that consumes Xamarin Essentials NuGet package.
2. Call `await CheckStatusAsync()` for the `Microphone` permission class.

### Expected Behavior
Should return `true` or `false`

### Actual Behavior
Throws `PermissionException`

### Basic Information

- Version with issue: 1.5.2
- Last known good version: NA
- IDE: VS 2019 16.5.2
- Platform Target Frameworks: 
  - UWP:  17763 
- Affected Devices: Windows Desktop

### Note
I recognize in the source code file that capability is checked by XML parsing the generated _AppxManifest.xml_ file, [here](https://github.com/xamarin/Essentials/blob/develop/Xamarin.Essentials/Permissions/Permissions.uwp.cs#L24). Maybe I am misinterpreting, but should not the namespace prefix be *f:* or empty rather than *x:* in the select element calls? According to [this](https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/what-s-changed-in-windows-10) document at least, you would get that impression.

**Comment by jamesmontemagno** [comment]
So, the default namespace is x: as that is the default. 

```
<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" IgnorableNamespaces="uap mp">
 ```

And the device capability should be:
```
    <DeviceCapability Name="microphone"/>
```

Can you copy/paste parts of your manifest.

**Comment by jamesmontemagno** [comment]
Looks like it does repro here. Putting some tests in place! Thanks!

For now you could probably skip it on UWP if you know it is in the manifest. That is all that it checks.

**Comment by anders9ustafsson** [comment]
Just for the sake of argument :-) , this is the capabilities section of my _Package.appxmanifest_ file:

>  <Capabilities>
    <Capability Name="privateNetworkClientServer" />
    <Capability Name="internetClientServer" />
    <uap:Capability Name="enterpriseAuthentication" />
    <uap:Capability Name="picturesLibrary" />
    <DeviceCapability Name="webcam" />
    <DeviceCapability Name="microphone" />
    <DeviceCapability Name="location"/>
  </Capabilities>

When you say "skip", do you mean that I should not execute `CheckStatusAsync` for UWP? It would be a little awkward, since the code is currently in a .NET Standard assembly, shared by UWP, iOS and Android apps. It is not a showstopper for me at the moment, but I hope the issue gets fixed in the near future. Thanks for looking further into the issue!

**Comment by jamesmontemagno** [comment]
So the "CheckStatus" on UWP for microphone jsut checks that you have it in there.

You could just do:

```
var checkStatus = DeviceInfo.Platform != DevicePlatform.UWP;
if (checkStatus)
{
 var status = await Permissions.CheckStatusAsync<Permissions.Microphone>();
}
```

Since you add it in you really don't need to check it as there isn't additional checks in place and it doesn't request microphone access currently at all. If you want you could grab the nuget from: https://dev.azure.com/xamarin/public/_build/results?buildId=16137&view=artifacts&type=publishedArtifacts

**Comment by jamesmontemagno** [comment]
Closed and in 1.5.3 release upcoming
